src/config/musl.md

```diff
@@ -0,0 +1,67 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms (although binary packages are not available for i686). Additionally,
+all compatible packages in our official repositories are available with
+musl-linked binaries in addition to their glibc counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+## Possible problems
+
+In some cases, software compatibility may be worse with musl.
+
+Proprietary software rarely supports non-glibc libc implementations, although
+sometimes these applications are available as [flatpaks](https://flatpak.org/),
+which provide their own libc in the image.
+
+### glibc chroot
+
+Software requiring glibc can be run in a glibc chroot.
+
+Create a directory that will contain the chroot, and install a base system in it
+via the `base-voidstrap` package. If network access is required, copy
+`/etc/resolv.conf` and `/etc/hosts` into the chroot.
+
+Several directories then need to be mounted as follows:
+
+```
+# mount -t proc none <chroot_dir>/proc
+# mount -t sysfs none <chroot_dir>/sys
+# mount --rbind /dev <chroot_dir>/dev
+# mount --rbind /run <chroot_dir>/run
+```
+
+Use [chroot(1)](https://man.voidlinux.org/chroot.1) to change to the new root,
+then run glibc programs as usual. Once you've finished using it, unmount the
+chroot using [umount(8)](https://man.voidlinux.org/umount.8).
+
+To seamlessly use graphical applications, install the `xhost` package and run:
```

Comment:
I'm still not sure this is a complete solution? Wouldn't you at least need to export a DISPLAY variable?

Reply:
Fair question! That seems like a distinct possibility; this was information i merely ported from the wiki. Should i test this in my musl Void VM?

Reply:
I'm inclined to think we can just skip this, and maybe come back later.

Reply:
Removed.